File: Wp.CIS.LynkSystems.Model/Administrative/AuditHistoryModel.cs
using System;
using System.Collections.Generic;
using System.Text;
using Wp.CIS.LynkSystems.Model.Enums;

namespace Wp.CIS.LynkSystems.Model.Administrative
{
    public class AuditHistoryModel
    {
        public LidTypeEnum LidType { get; set; }

        public int Lid { get; set; }

        public ActionTypeEnum ActionType { get; set; }

        public DateTime ActionDate { get; set; }

        public string UserName { get; set; }

        public string Notes { get; set; }

        public int AuditId { get; set; }
    }
}

File: Wp.CIS.LynkSystems.Web/ClientApp/app/models/administration/product/productmaintenance/audit-history.model.ts

import { ActionTypeEnum } from '../../../common/action-type.enum';
import { LidTypesEnum } from '../../../common/lid-types.enum';

export class AuditHistoryModel {

    public lidType: LidTypesEnum;

    public lid: number;

    public actionType: ActionTypeEnum;

    public actionDate: Date;

    public userName: string;

    public notes: string;

    public auditId: number;

}